test/e2e/windows/kubelet_stats.go

```diff
@@ -77,12 +77,20 @@ var _ = SIGDescribe("[Feature:Windows] Kubelet-Stats [Serial]", func() {
 						}
 						statsChecked = statsChecked + 1
 
-						framework.ExpectEqual(*podStats.CPU.UsageCoreNanoSeconds > 0, true, "Pod stats should not report 0 cpu usage")
-						framework.ExpectEqual(*podStats.Memory.WorkingSetBytes > 0, true, "Pod stats should not report 0 bytes for memory working set ")
-
+						if *podStats.CPU.UsageCoreNanoSeconds <= 0 {
+							framework.Failf("Pod %s/%s stats report cpu usage equal to %v but it should be greater than 0", podStats.PodRef.Namespace, podStats.PodRef.Name,*podStats.CPU.UsageCoreNanoSeconds)
+						}
+						if *podStats.Memory.WorkingSetBytes <= 0 {
+							framework.Failf("Pod %s/%s stats report bytes for memory working set equal to %v but it should be greater than 0", podStats.PodRef.Namespace, podStats.PodRef.Name, *podStats.Memory.WorkingSetBytes)
+						}
 						for _, containerStats := range podStats.Containers {
-							framework.ExpectEqual(containerStats.Logs != nil, true, "Pod stats should have container log stats")
+							if containerStats.Logs == nil {
+								framework.Failf("Pod %s/%s stats should have container log stats, but it is equal to %v", podStats.PodRef.Namespace, podStats.PodRef.Name,nil)
```

Comment:
There's no point in passing a nil as parameter. Just put that into the message and also make it shorts ("but it is nil").